project/docs/gen_credits.py.jinja

```diff
@@ -40,6 +41,8 @@ def get_credits_data() -> dict:
         "indirect_dependencies": sorted(indirect_dependencies),
         {%- if author_username == "pawamoy" %}
         "more_credits": "http://pawamoy.github.io/credits/",
+        {%- else %}
+        "more_credits": more_credits,
         {%- endif %}
```

Comment:
Lets deal with both cases in a single line:

```suggestion
        "more_credits": "{% if author_username == "pawamoy" %}http://pawamoy.github.io/credits/{% endif %}",
```

Reply:
Done single-liner :-)

Reply:
Thanks! Nitpick though, you reversed the condition and used single-quotes, I'd like the opposite: `{% if author_username == "pawamoy" %}http://pawamoy.github.io/credits/{% endif %}`

Reply:
Done!